pkg/kubecost/asset.go

```diff
@@ -479,7 +480,7 @@ func (ca *Cloud) SetAdjustment(adj float64) {
 
 // TotalCost returns the Asset's total cost
 func (ca *Cloud) TotalCost() float64 {
-	return ca.Cost + ca.adjustment
+	return ca.Cost + ca.adjustment + ca.Credit
```

Comment:
What unit is credits in? positive currency or negative?  Can you make a note on the Credit field?